media-sound/snapcast/snapcast-9999.ebuild

```diff
@@ -1,32 +1,33 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
```

Comment:
You're keeping the user eclass? Shock!

Reply:
Ah, no :) I was waiting for someone to take a look at this first. I guess I'll have to make new acct packages to drop the eclass, that's what the CI is complaining about...

Reply:
Pushed new acct* packages

Reply:
side-note: I just saw now that new users and groups should be sent to gentoo-dev ML for review. I will do that in the following days.